.github/workflows/ingestion_client.yaml

```
@@ -53,17 +53,17 @@ jobs:
         run: dotnet publish -c Release -f net8.0 --no-self-contained --output ./StartTranscriptionByTimer samples/ingestion/ingestion-client/StartTranscriptionByTimer/StartTranscriptionByTimer.csproj
 
       - name: Zip StartTranscriptionByTimer
-        run: cd StartTranscriptionByTimer && zip -r ../StartTranscriptionByTimer.zip *
+        run: cd StartTranscriptionByTimer && zip -r ../StartTranscriptionByTimer.zip * .azurefunctions
```

This should work as well (also below):

```suggestion
        run: cd StartTranscriptionByTimer && zip -r ../StartTranscriptionByTimer.zip .
```

(The glob `*` is interpreted by the Shell below calling `zip`. Default is not to expand hidden files; e.g., for Bash, `man bash` -> Pathname Expansion)

---

Fixed!